i18n/language/language.go

```diff
@@ -17,23 +19,35 @@ type Language struct {
 	*PluralSpec
 }
 
+// NewLanguage returns new Language with specified tag and spec.
+func NewLanguage(tag string, spec *PluralSpec) *Language {
```

Comment:
Actually, there is a problem when we want to create `&Language{}` inside the package.
E.g. having `&Language{tag, spec}` compiler complains: 
```
i18n/language/language.go:58: cannot use tag (type string) as type []string in field value
i18n/language/language.go:58: cannot use spec (type *PluralSpec) as type string in field value
```
`NewLanguage` solves it.